packages/mdc-button/_mixins.scss

```diff
@@ -381,6 +381,15 @@ $query: mdc-feature-all()) {
   }
 }
 
+///
+/// Sets the button label to overflow as ellipsis
+///
+@mixin mdc-button-label-overlow-ellipsis($query: mdc-feature-all()) {
```

Comment:
When do we use this mixin? are there any special cases? Can we apply this mixin to MDC button by default?

Reply:
This was requested by an internal team.